src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,20 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.TextUi;
+
+public abstract class Command {
+    private final boolean isExit;
+
+    public Command (boolean isExit) {
```

Comment:
Minor change needed here.

```suggestion
    public Command(boolean isExit) {
```